.gitignore

```diff
@@ -11,4 +11,3 @@ gsutil_*.zip
 .metadata/*
 .idea/*
 .vscode/*
-venv*/*
```

Comment:
Why remove the venv directory? It's nice to hide repo-specific virtualenv dirs from git.

Reply:
oh, good point, must have been a weird merge thing.... I'll add it back in.